invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -50,6 +57,25 @@ function InVideoQuizXBlock(runtime, element) {
         }
     }
 
+    function getVideoDimensions() {
+        videoPosition = $('.tc-wrapper', video).position().top;
+        videoHeight = $('.tc-wrapper', video).css('height');
+        videoWidth = $('.tc-wrapper', video).css('width');
+        return [videoPosition, videoHeight, videoWidth];
```

Comment:
Also, looking at how this data is actually used, I suspect a dictionary/object may be a more apt data structure.

```javascript
return {
    'top': position,
    'height': height,
    'width': width
};
```

That means when we're modifying the CSS below, it's as easy as:

```javascript
currentProblem.css(videoDimensions);
```

Of course, the `for` loop will need updated, but that should be simple enough:

```javascript
for (var key in knownVideoDimensions) {
    if (knownVideoDimensions.hasOwnProperty(key)) {
        if (knownVideoDimensions[key] !== currentVideoDimensions[key]) {
            return true;
        }
    }
}
return false;
```

This makes the code more obvious, by avoiding directly indexing into the array `arr[0]`, `arr[1]`, etc., as that's opaque.